driver-core/src/main/com/mongodb/operation/ChangeStreamBatchCursor.java

```diff
@@ -23,64 +23,68 @@
 import com.mongodb.ServerCursor;
 import com.mongodb.binding.ReadBinding;
 import org.bson.BsonDocument;
+import org.bson.BsonTimestamp;
 import org.bson.RawBsonDocument;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import static com.mongodb.operation.ChangeStreamBatchCursorHelper.isRetryableError;
 
-final class ChangeStreamBatchCursor<T> implements BatchCursor<T> {
+final class ChangeStreamBatchCursor<T> implements ChangeStreamCursor<T> {
     private final ReadBinding binding;
     private final ChangeStreamOperation<T> changeStreamOperation;
 
-    private BsonDocument resumeToken;
-    private BatchCursor<RawBsonDocument> wrapped;
+    private BsonTimestamp startAtOperationTime;
```

Comment:
If I understand this variable correctly, maybe it's better to rename it to `initialStartAtOperationTime`?

Reply:
I have renamed the variable.